package.json

```diff
@@ -67,6 +67,12 @@
         "category": "ONE",
         "icon": "$(refresh)"
       },
+      {
+        "command": "onevscode.create-cfg",
+        "title": "Create new ONE configuration",
```

Comment:
There are people who have _no_ idea what ONE configuration file is. So I think this title is very important for them. :-)
They must understand why they need to create cfg file by looking at this title.

How about adding 'why' message, for example., like the following?

"Create a new ONE config file for compilation, execution, or profiling"

Reply:
Well I personally prefer the simple message.

I think we need to inform the users more properly if they don't know about ONE config - as ONE config is kinda a key notion for our tool. (It's not just a config but it's kidna master file, such as CMakeLists.txt.)

**Instead, I'd like to suggest: (1) or (2)**

#### (1) notification for the users who don't have ANY config file.
For those user who don't have idea about ONE config, we can provide vscode notification saying: You don't have any ONE configuration in your workspace. Try create one by clicking + next to your model files. [reference:notification](https://code.visualstudio.com/api/ux-guidelines/notifications)

#### (2) welcome view
Or we can provide welcome view for more details.
[welcome view](https://code.visualstudio.com/api/ux-guidelines/views#welcome-views)

Reply:
I will add those two things as TODOs at #584

Reply:
Those two ideas are great. 😸 

I think tooltip is also important because small percentage of users may not remember what they saw in welcome view or notification.

If you prefer shorter sentence, how about the following?

- `Create a config file for compilation`

Reply:
I am not sure ... I don't think that icon tooltip should provide that much information.

Reply:
>  then the naming 'ONE configuration' is weak

I agree. That is why I didn't use `ONE configuration file` but `config file`.
I think people like me will think this tooltip helpful.

If you don't like this approach, please ignore the above comment and continue without my approval.

Reply:
@hyunsik-yoon I appreciate your understanding. We can discuss about it later on. 

> I agree. That is why I didn't use ONE configuration file but config file. 

I personally think the both "ONE configuration" and "config file" are weak, 

I thought the naming should be somthing like
(naming) 'config file' ->  'onecc file' 
(extension) '.cfg' -> '.one' 

Such like that, to let users know that :
>  ONE config is kinda a key notion for our tool. (It's not just a config but it's kidna master file, such as CMakeLists.txt of cmake.)

Reply:
I nowadays think that UI is too difficult.
I am a person who could represents only a small portion of various customers.
So I think that usability test in future will be very important for various customers.